joml-geometry/src/main/java/org/terasology/joml/geom/AABBfc.java

```diff
@@ -89,6 +90,38 @@
      */
     Vector3d center(Vector3d dest);
 
+    /**
+     * Expand this aabb by adding the given {@code extents} for each face of the region.
+     *
+     * @param dx the amount to extend this aabb by the x axis in both directions
+     * @param dy the amount to extend this aabb by the y axis in both directions
+     * @param dz the amount to extend this aabb by the z axis in both directions
+     * @param dest will hold the result
+     * @return {@code dest} (after modification)
+     * @throws IllegalArgumentException if extending this region would result in any non-positive dimension
+     */
+    AABBf expand(float dx, float dy, float dz, AABBf dest);
```

Comment:
This _could_ be implemented right here as a default method. This means one class less in case anything changes/no double maintenance...